extensions/wikia/EditPageLayout/js/plugins/FlowTracking.js

```diff
@@ -0,0 +1,30 @@
+(function(window,$){
+
+	var WE = window.WikiaEditor = window.WikiaEditor || (new Observable());
+
+	WE.plugins.flowtracking = $.createClass(WE.plugin,{
+
+		initEditor: function( editor ) {
+			require(['wikia.flowTracking', 'wikia.querystring', 'mw'], function(flowTrack, QueryString, mw) {
+
+				// Track only creating articles from namespace 0 (Main)
+				// IMPORTANT: on Special:CreatePage even after providing article title the namespace is set to -1 (Special Page)
+				if (mw.config.get('wgNamespaceNumber') === 0 && mw.config.get('wgArticleId') === 0) {
```

Comment:
what does wgArticleId cover in this condition? The above comment does not explain that